.github/workflows/ci.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
       run: go test -v ./...
 
     - name: Build Executable
-      run: go build -o cmd/aterm/aterm cmd/aterm/*.go
+      run: go build -ldflags '-X main.gitHash=${{ needs.store-sha8.outputs.sha8 }}' -o cmd/aterm/aterm cmd/aterm/*.go
```

Comment:
This also needs to fill in the tag as well (assuming it's a tagged build).

Reply:
Yeah, this PR was more to handle just getting the info into the system. I can merge it into the PR we have for displaying updates, in addition to just rendering this info somewhere, either behind a flag, or as part of the startup. 

I'll look into  go-info to see if it's an easy fit into the project (I'm sure it is)

Reply:
I don't know that we need to merge this into the PR that adds updates but I would like to have a single PR that handles adding version + runtime information and displaying it. The update PR will need this information in place in order to accurately request updates.